test/integration/targets/hcloud_floating_ip_info/aliases

```diff
@@ -1,3 +1,2 @@
 cloud/hcloud
 shippable/hcloud/group1
-disabled # See: https://github.com/ansible/ansible/issues/62414
```

Comment:
This should be done in a separate PR so it can be backported to the stable branches which also disabled the test.